cmd/influxd/server_integration_test.go

```diff
@@ -726,7 +726,7 @@ func TestClientLibrary(t *testing.T) {
 		}
 
 		if test.query.Command != "" {
-			time.Sleep(50 * time.Millisecond)
+			time.Sleep(100 * time.Millisecond)
```

Comment:
Yes, 50 was sometimes coming up short.  100 always worked.  Until we get our wait endpoint back up to par this feels like the best we can do.